provision/vvv-init.sh

```diff
@@ -159,11 +159,16 @@ else
   sed -i "s#{{LIVE_URL}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 fi
 
-get_config_value 'wpconfig_constants' |
+shyaml get-values-0 "sites.${VVV_SITE_NAME}.custom.wpconfig_constants" < "${VVV_CONFIG}" |
```

Comment:
Is  there no `get_config_values`?

Reply:
No seems that we don't have it, we are doing similar in https://github.com/Varying-Vagrant-Vagrants/VVV/blob/e69df501960cad435c1680027aac3054fb52d487/provision/provision-site.sh#L44